### ISSUE TITLE
Export to Gist Failed

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#1138]
File I am working on:
[stripesExample.zip](https://github.com/nortikin/sverchok/files/710024/stripesExample.zip)

I think the issue happens after I zip and then want to export to gist. no idea why.

https://gist.github.com/enzyme69/00a2e6ac2486633524df446da7c9b1c3

**Comment by enzyme69** [comment]
This also failed:
[sv_stripes_demo_003_2017_01_17_05_33.zip](https://github.com/nortikin/sverchok/files/710082/sv_stripes_demo_003_2017_01_17_05_33.zip)

**Comment by zeffii** [comment]
This has nothing to do with zipping first. The same problem for both .blends , the obj_in node.

![image](https://cloud.githubusercontent.com/assets/619340/22010375/1bb3d21a-dc89-11e6-8cba-18af044dcdd5.png)

The error message is quite telling..
```
raise TypeError(repr(o) + " is not JSON serializable")
```

`obj_in` is not, and has never been, supported by iosjon, because it references scene objects. -- obj_in_mk3 (and mk1, mk2) don't store scene data (ie, don't serialize the data) Maybe i'm losing my mind, but i'm pretty sure this has been discussed before. 

##What can you do?
best you can do at the moment is unselect the scene object, then press 'Get Selection' so it doesn't store references.  https://gist.github.com/16899fc80cafd8efd1279de67f6f9605

##What can i do?
I know this isn't a neat solution, but i might include an automatic `unload objects' in iojson export code,, so it doesn't attempt to serialize. Or include a skip in the export code to just bypass object references.

---------
The bottom line is. If you use obj_in and it requires an object, then share blends with . For most other scenarios iojson will likely work.

**Comment by zeffii** [comment]
i will address this today at some point in the export code.

**Comment by zeffii** [comment]
### Update

this part of iojson was corrected. The exporter doesn't care if objects are references. as per the old way. 

### Open issue
Objects referenced in the current scene/layout will not automatically be stored in the json.  While this is not too hard to add, it does take time, that can be avoided by sharing .blend instead ( blend will give smaller file sizes with larger objects anyway... but i'm not sure what the break even point is.. so ..effectively i don't care )
